### PR TITLE
[MM-28140] e2e/system_console/customization: add T1030

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Add T1030: Custom branding removed - Default site name and description, and brand image and text - false

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28140

### CODE REVIEW

**Comment by lindy65** [comment]
Hi @mgdelacroix,

Can you please change the title of the test to indicate `Default` site name?  This is because a blank site name can no longer be saved so the current title is misleading.  (I've updated the existing test in TM4J to indicate the updated title)

Thanks!

**Comment by mgdelacroix** [comment]
@lindy65 done! :)

**Review by lindy65 — APPROVED** [review]
Thanks @mgdelacroix - looks good to me 👍

**Review by isacikgoz — APPROVED** [review]
LGTM 🚀

File: e2e/cypress/integration/system_console/site_configuration/customization_spec.js [hunk 1]
```diff
@@ -249,6 +249,45 @@ describe('Customization', () => {
...
+        cy.url().should('include', '/login');
+
+        // * Ensure Site Name and Description have the default values
+        cy.get('#site_name').should('have.text', siteName);
+        cy.get('#site_description').should('have.text', 'All team communication in one place, searchable and accessible anywhere');
+
+        // * Ensure that the custom branding img is not visible
+        cy.get('img').should('not.be.visible');
+
+        // * Ensure tha the custom branding text is empty
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/site_configuration/customization_spec.js:288]:
```suggestion
        // * Ensure that the custom branding text is empty
```

**Review by saturninoabril — APPROVED** [review]
Thanks @mgdelacroix, LGTM! Left suggestion on typo.

**Pull request merged**